gapic.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CMD="$0"
+
+# Set variables used by this script.
+# All of these are set in options below, and all but $PROTO_PATH are required.
+IMAGE=
+IN=
+OUT=
+PLUGIN_OPTIONS=
+PROTO_PATH=`pwd`
```

Comment:
I've been using `$(pwd)` rather than `pwd`. I believe there's some difference, but I don't remember it.

Reply:
This file is mostly copy/paste from go and python versions, so I'd rather leave this as-is so it's as similar to the sources as possible.

Reply:
Sure, that's fine.